packages/flutter_tools/templates/plugin/ios-objc.tmpl/projectName.podspec.tmpl

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.platform = :ios, '8.0'
+  s.platform = :ios, '9.0'
```

Comment:
Note these newly created plugins will not be able to be used by Flutter apps with a minimum of 8.0 (compilation time error), so that's all apps currently in the wild now.

I really wish there wasn't an error for this, or at least I wish it was suppressible.

I'm willing to be convinced we shouldn't increase the plugin minimum requirement for a few stable cycles until more apps have upgraded to 9.0...

Reply:
so you're saying an app can't use a plugin with a higher minimum ios version?

Reply:
Yes. I'm going to revert the plugin min version bump, it seems like we need to talk about that separately.

Reply:
SGTM